### PR TITLE
182_update_forest_language

### PR DESCRIPTION
**Pull request by Dmac26**
﻿## Summary
Addresses issue:
https://app.zenhub.com/workspaces/open-forest-58f8bbe105a35fad2d275a0c/issues/usdaforestservice/usfs-timber-permitting/182

This PR updates the existing language for all forests, refining the text to match the needs of each individual forest.

## This pull request is ready to merge when...
- [ ] Feature branch starts with the issue number
- [ ] Is connected to its original issue via zenhub 👇
- [ ] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [ ] This code has been reviewed by someone other than the original author﻿## Summary

### CODE REVIEW

**Comment by mgwalker** [comment]
Is this one I should review? Tag me if so!

File: frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/when-to-cut/when-to-cut.component.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h3>When to cut firewood</h3>
   <p>Firewood permits are valid through the calendar year.</p>
   <p>See <a href="#prohibited">prohibited areas and restrictions</a> for more information.</p>
   <h3>Cost</h3>
-  <p>Firewood permits cost <strong>$20.00 for 6 cords of wood</strong>. All permits are nonrefundable</p>
+  <p>Firewood permits cost <strong>$20.00 for 6 cords of wood</strong>. Limit of (waiting on the number from POC) cords per household per year. All permits are nonrefundable</p>
```
Comment (mgwalker) [inline frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/when-to-cut/when-to-cut.component.html:39]: Similar question re: parenthetical. If we don't have this information yet, is there an issue to come back and fill it in?

File: frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/how-to-cut/how-to-cut.component.html [hunk 2]
```diff
@@ -23,41 +26,55 @@ <h3>What to expect</h3>
   <p>Before you go check the weather forecast and always let someone know where you will be and when you are expected to return</p>
   <p>Forest Service roads may have heavy logging traffic. Stay alert and give log truck and equipment plenty of space.</p>
   <p>Park in areas so that traffic can get by safely and do not block gates.</p>
-  <h3>Using your Firewood</h3>
-  <p>Gathering of campwood is allowed without a permit as long as the wood is burned at your camp. A permit is required to transport any unused firewood home.</p>
+  <p>Drive safely and be alert for winter driving conditions. Your vehicle should be equipped with 4-wheel drive, snow tires, chains, and a shovel during the winter season. (placement of this one moved).</p>
```
[the remark this answers is not in the document]
Comment (Dmac26) [reply frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/how-to-cut/how-to-cut.component.html]: Thanks for catching that. Removed!

File: frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/when-to-cut/when-to-cut.component.html (hunk 1 above)
Comment (Dmac26) [reply frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/when-to-cut/when-to-cut.component.html]: Removed this one too.

**Pull request merged**